airbyte-workers/src/test/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowTest.java

```diff
@@ -1324,119 +1276,6 @@ void testCanRetryFailedActivity() throws InterruptedException {
 
   }
 
-  @Nested
-  @DisplayName("Test workflow where the child workflow throw a hearbeat timeout exception")
-  class HeartbeatFailureWorkflow {
-
-    @BeforeEach
-    public void setup() {
-      setupSpecificChildWorkflow(SyncWorkflowFailingWithHearbeatTimeoutException.class);
-    }
-
-    @ParameterizedTest
-    @MethodSource("io.airbyte.workers.temporal.scheduling.ConnectionManagerWorkflowTest#getMaxAttemptForResetRetry")
-    public void failedResetContinueAttemptAsReset(final int maxAttempt) throws InterruptedException {
```

Comment:
I removed this block from the tests because it was testing a specific case where a failed reset needed to set a special flag to indicate that the next job should also be a reset. This is no longer necessary because the Job creation logic decides whether or not to run a reset based on what is in the stream_resets table. So these tests are also no longer necessary.